streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -126,7 +126,7 @@ public Builder(final String storeName, final Serde<K> keySerde, final Serde<V> v
         }
 
         @Override
-        public StateStore build() {
+        public InMemoryTimeOrderedKeyValueBuffer<K, V> build() {
```

Comment:
Nice catch!

Reply:
Thanks! I was chasing down an unchecked cast and was pleasantly surprised that it was totally unnecessary!